include/aspect/postprocess/interface.h

```diff
@@ -378,6 +401,46 @@ namespace aspect
       return NULL;
     }
 
+    /**
+     */
+    template <int dim>
+    template <typename PostprocessorType>
+    inline
+    bool
+    Manager<dim>::has_postprocessor_of_type () const
+    {
+      for (typename std::vector<std_cxx11::shared_ptr<Interface<dim> > >::const_iterator
+           p = postprocessors.begin();
+           p != postprocessors.end(); ++p)
+        if (Utilities::object_is_of_type<PostprocessorType>(*(*p)))
+          return true;
+
+      return false;
+    }
+
+    /**
+     */
+    template <int dim>
+    template <typename PostprocessorType>
+    inline
+    PostprocessorType &
+    Manager<dim>::get_postprocessor_of_type () const
+    {
+      AssertThrow(has_postprocessor_of_type<PostprocessorType> (),
+                  ExcMessage("You asked Postprocess:Manager::get_postprocessor_of_type() for a "
+                             "postprocessor type that could not be found in the current model. Activate this "
```

Comment:
I think you can print the actual type of the requested postprocessor using `typeid(PostprocessorType).name()`. You'll have to demangle this, though. I think there is a function for this in `dealii::Utilities`.